FinancialTransactionsApi/V1/Helpers/GeneralModels/PaginatedResponse.cs

```diff
@@ -1,8 +1,10 @@
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using FinancialTransactionsApi.V1.Boundary.Response;
 
 namespace FinancialTransactionsApi.V1.Helpers.GeneralModels
 {
+    [ExcludeFromCodeCoverage]
```

Comment:
Why are we excluding the Pagination classes from test coverage?

Reply:
Done